default.paths.json

```diff
@@ -0,0 +1,14 @@
+{
+  "digitalLandFrontendPath": "node_modules/digital-land-frontend/",
+  "govukFrontendPath": "node_modules/govuk-frontend/",
+  "scssPath": "./src/scss",
+  "templatesPath": "'application/templates'",
+  "staticFilesPath": "'application/static'",
+  "watchPaths": "'./src/scss/**/*.scss' './src/javascripts/**/*.js'",
+  "stylesheetsOutputPath": "assets/stylesheets",
+  "jsOutputPath": "assets/javascripts",
+  "imagesOutputPath": "assets/images",
+  "govukOutputPath": "assets/",
+  "rollupConfig": "./rollup.config.js",
+  "serverURL": "http://localhost:5000"
+}
```

Comment:
configuration object used by nps taken from dl-frontend. imagine there's redundant config in here. but I think that should be picked up as part of another ticket